cmd/kubeadm/app/cmd/config.go

```diff
@@ -160,19 +163,31 @@ func getAllAPIObjectNames() []string {
 
 func getDefaultedInitConfig() (*kubeadmapi.InitConfiguration, error) {
 	return configutil.ConfigFileAndDefaultsToInternalConfig("", &kubeadmapiv1alpha3.InitConfiguration{
-		API:               kubeadmapiv1alpha3.API{AdvertiseAddress: "1.2.3.4"},
-		BootstrapTokens:   []kubeadmapiv1alpha3.BootstrapToken{sillyToken},
-		KubernetesVersion: fmt.Sprintf("v1.%d.0", constants.MinimumControlPlaneVersion.Minor()+1),
+		// TODO: Probably move to getDefaultedClusterConfig?
```

Comment:
?

Reply:
I think, this one was a comment in the old PR. We decided to mark all unaddressed comments in the original PR as TODOs in the new one.